docs/deprecated.md

```diff
@@ -0,0 +1,40 @@
+# Deprecated functionality
+
+Deprecated features are features that lightweightM2M-iotagent stills support but that are
+not maintained or evolved any longer. In particular:
+
+-   Bugs or issues related with deprecated features and not affecting
+    any other feature are not addressed (they are closed in github.com
+    as soon as they are spotted).
+-   Documentation on deprecated features is removed from the repository documentation.
+    Documentation is still available in the documentation set associated to older versions
+    (in the repository release branches).
+-   Deprecated functionality is eventually removed from lightweightM2M-iotagent. Thus you
+    are strongly encouraged to change your implementations using lightweightM2M-iotagent
+    in order not rely on deprecated functionality.
+
+A list of deprecated features and the version in which they were deprecated follows:
+
+* Support to Node.js v4 in lightweightM2M-iotagent 1.0.0. The use of Node.js v8 is highly recommended.
```

Comment:
NGSIv1 should be also added to this list, to be coherent with the table at the bottom.

Reply:
Fixed with 41edfcdf6b4da736051e4894ce69f5622376da9a